app/controllers/hyrax/dashboard/nest_collections_controller.rb

```diff
@@ -45,21 +45,21 @@ def create_relationship_under
 
         def build_within_form
           child = Collection.find(params.fetch(:child_id))
-          authorize! :edit, child
+          authorize! :read, child
```

Comment:
My understanding of collections is, since we are editing the child's relationships, we should have :edit permissions. I also believe there is a tension in the direction of the relationship. 

In the case where the Library has deposited a "Featured Work of Utmost Awesomeness" and a user creates a collection "Silly Jokey Stuff" and includes the "Featured Work of Utmost Awesomeness" in that collection, then on the work there would be an asserted relationship to "Silly Jokey Stuff". I don't believe that would be an acceptable relationship in the eyes of the Library.

I'm not certain how we should proceed with this.

Reply:
Based on Slack conversations, I'm fine proceeding with this implementation. I'm not certain how we could do otherwise given the current implementation constraints.